### PR TITLE
Add firebase Id in user schema and missing fields of courseDetails.txt

### PR DESCRIPTION
**Pull request by Devesh21700Kumar**
**Description**

Add firebase Id in user schema and response body

Terminal commands to test code:-
1). npm run lint
2). npm run lint:fix
3). nest start
4). npm run test

### CODE REVIEW

**Review by Devesh21700Kumar — COMMENTED** [review]
@Abhishek-kumar09 Added fId feild in userSchema and added some feilds missing as per courseDetails.txt

File: src/modules/course/docUtils/course.responsedoc.ts [hunk 1]
```diff
@@ -195,6 +201,13 @@ export class ScheduleResponseBody {
    */
 
   time: string;
+
+  /**
+   * The lecture Number/Index among all lectures in the schedule
+   * @example 8
+   */
+
+  lectureNumber: number;
```
Comment (Abhishek-kumar09) [inline src/modules/course/docUtils/course.responsedoc.ts:210]: What is this?

Comment (Devesh21700Kumar) [reply src/modules/course/docUtils/course.responsedoc.ts:210]: missing field frim courseDetails.txt

Comment (Abhishek-kumar09) [reply src/modules/course/docUtils/course.responsedoc.ts:210]: Videosnum is not sufficient?

Comment (Devesh21700Kumar) [reply src/modules/course/docUtils/course.responsedoc.ts:210]: this is for the schedule
the index for every lecture

Comment (Abhishek-kumar09) [reply src/modules/course/docUtils/course.responsedoc.ts:210]: How a number can store index of all the scheduled courses? @Devesh21700Kumar

Comment (Devesh21700Kumar) [reply src/modules/course/docUtils/course.responsedoc.ts:210]: > How a number can store index of all the scheduled courses? @Devesh21700Kumar

Its a field of schedule schema so every schedule will store its *own* index only.

One object will have 1
another will have 2 
another object will have 3

**Pull request merged**